### PR TITLE
feat(notifier): adds notifier to meta repo

### PR DESCRIPTION
**Pull request by vjgene**
resolves: https://rsklabs.atlassian.net/jira/software/projects/RMKT/boards/60?selectedIssue=RMKT-842

### CODE REVIEW

File: scripts/createConfig.js [hunk 1]
```diff
@@ -68,6 +68,17 @@ networks.forEach(network => {
       notifierConfig[network] = JSON.parse(fs.readFileSync(file));
   }
 
+// RIF Notifier config
+  if (args.includes('rif-notifier')) {
```
Comment (0xartem) [inline scripts/createConfig.js]: The `notifier` is already present, you don't need to add a new one. If the `notifier` is passed as an argument - you just run the script

File: scripts/createConfig.js [hunk 2]
```diff
@@ -68,6 +68,17 @@ networks.forEach(network => {
...
+    const notifierConfigPath = "./notifier/config/config-";
+    const file = notifierConfigPath + network + ".json";
+    if (fs.existsSync(file))
```
Comment (0xartem) [inline scripts/createConfig.js]: there is a curly brace missing?

File: package.json [hunk 3]
```diff
@@ -45,12 +46,15 @@
     "cache": "npm run cache:config && npm run cache:start",
     "rns-manager:start:pm2": "cd rns-manager && REACT_APP_URL=http://localhost:3001/ pm2 start npm --name=rns-manager -- start && cd ..",
     "rns-manager": "cd rns-manager && REACT_APP_URL=http://localhost:3001/ npm start && cd ..",
+    "notifier:config": "cd notifier && bin/config.sh",
+    "notifier:start:pm2": "cd notifier && pm2 start --image-name notifier 'notifier-prov-cli start'",
+    "notifier:start": "cd notifier && notifier-prov-cli start",
```
Comment (0xartem) [inline package.json]: add `cd ..` in the end to return back to the root folder

Comment (0xartem) [reply package.json]:
```suggestion
    "notifier:start": "cd notifier && notifier-prov-cli start && cd ..",
```

File: package.json [hunk 4]
```diff
@@ -45,12 +46,15 @@
     "cache": "npm run cache:config && npm run cache:start",
     "rns-manager:start:pm2": "cd rns-manager && REACT_APP_URL=http://localhost:3001/ pm2 start npm --name=rns-manager -- start && cd ..",
     "rns-manager": "cd rns-manager && REACT_APP_URL=http://localhost:3001/ npm start && cd ..",
+    "notifier:config": "cd notifier && bin/config.sh",
+    "notifier:start:pm2": "cd notifier && pm2 start --image-name notifier 'notifier-prov-cli start'",
```
Comment (0xartem) [inline package.json]:
```suggestion
    "notifier:start:pm2": "cd notifier && pm2 start notifier-prov-cli --name notifier -- start && cd ..",
```

File: package.json [hunk 5]
```diff
@@ -23,7 +24,7 @@
     "contracts:redeploy:devnet": "meta exec \"npx truffle migrate --network rskdevnet --reset\" --include-only $npm_package_config_contracts",
     "contracts:deploy:testnet": "meta exec \"npx truffle migrate --network rsktestnet\" --include-only $npm_package_config_contracts",
     "contracts:deploy:mainnet": "meta exec \"npx truffle migrate --network rskmainnet\" --include-only $npm_package_config_contracts",
-    "contracts:create-config": "node scripts/createConfig.js rns storage notifier",
+    "contracts:create-config": "node scripts/createConfig.js rns storage notifier rif-notifier",
```
Comment (0xartem) [inline package.json]: rif-notifer can be removed

File: README.md [hunk 6]
```diff
@@ -339,10 +339,48 @@ You should see in logs when new Agreements are detected and pinned.
 
 ## 8. RIF Notifier service
 
+RIF Notifier is a service that listens to events on the blockchain and notifies its users of the blockchain events using one of the supported notification methods (API/SMS/EMAIL)
+
```
Comment (0xartem) [inline README.md:343]:
```suggestion
> RIF Notifier is automatically started via the main setup. If you want to run it in the terminal, execute `npm run notifier` or with pm2 `npm run notifer:start:pm2`. Other basic commands are available under `npm run notifer:*`. For the detailed setup, please refer to the repository.
```

**Review by 0xartem — CHANGES_REQUESTED** [review]

File: package.json (hunk 5 above)
Comment (vjgene) [reply package.json]: i wanted to separate rif-marketplace-notifier from rif-notifier. Anyways, no issues we can use the same parameter

**Review by 0xartem — APPROVED** [review]

**Pull request merged**